### ISSUE TITLE
Show number of total citations of an entry

### ISSUE THREAD

**Issue by ThiloteE** [JabRef/jabref#8954]
**Problem:**
There is metadata out there that JabRef is unable to capture (yet). One type of this data is the total number of an entry being cited.

**Describe the solution you'd like**

Show the number of total citations of an entry *somewhere*.

**Additional context**
- This issue is inspired by the following feature request in the forum: https://discourse.jabref.org/t/number-of-paper-citation-information/2616
- this issue here is a sub-issue of the more general issue of showing more citation-related diverse metadata, such as information about the journal, author and so on. A related issue would be #6189

**To do:**
- Implementation difficulties:
   - [ ] Discussion is necessary to decide, where exactly the total number of citations should show in JabRef. Screenshots would be good.
   - [ ] Prefered: Finding public sources that do not require an API key.
      - [ ] If no public sources can be found, implement a solution that works, if users insert their own personal API key.

**Comment by YesWeCandrew** [comment]
I'd like to investigate this further and work on this issue, can it be assigned to me? I'll be working with one other person so we should be able to get it done

**Comment by ThiloteE** [comment]
You are welcome. If you really want to, you can of course, but since you are new to GitHub, I assume this is your first or second Open Source project. We have explicitly collected [good first issues](https://github.com/JabRef/jabref/labels/good%20first%20issue) for newcomers and we also have [candidates for university projects](https://github.com/orgs/JabRef/projects/3). If you ever feel like you are stuck, because this issue here is unexpectedly difficult and hard to achieve, you could instead have a look at those. If you ever decide to stop working on this, please comment here, so that we know.

**Comment by ThiloteE** [comment]
As a general advice, check out https://github.com/JabRef/jabref/blob/main/CONTRIBUTING.md for a start. Also, https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace is worth having a look at. Feel free to ask if you have any questions here on GitHub or also at JabRef's [Gitter](https://gitter.im/JabRef/jabref) chat.

**Comment by koppor** [comment]
Please also read into existing code at https://github.com/JabRef/jabref/pull/7160.

**Comment by YesWeCandrew** [comment]
Thanks ThiloteE and koppor - I'll let you know if we have any issues and will investigate #7160.

**Comment by HoussemNasri** [comment]
The number of citations and references can be provided by the SematicScholar API. It should be fairly easy to integrate it within the Citations relations tab.

I'll make a good first issue of this once I'm done with https://github.com/JabRef/jabref-issue-melting-pot/issues/281

![image](https://github.com/JabRef/jabref/assets/21198231/d3c0aae6-d41c-4193-8268-d108b0d50486)

**Comment by ThiloteE** [comment]
Sure, if you believe it will be easy, a fast win and you feel motivated, go for it :-)

**Comment by HoussemNasri** [comment]
I suggest having the number of citations/references in the heading, like in ArXiv Bibliographic Explorer

![image](https://github.com/JabRef/jabref/assets/21198231/bcba508c-ffbe-4300-8558-1fb620def0de)

**Comment by Siedlerchr** [comment]
Isn't this resolved thanks to scite.ai?   Refs https://github.com/koppor/jabref/issues/375  and https://github.com/JabRef/jabref/pull/10617 
<img width="979" alt="grafik" src="https://github.com/user-attachments/assets/781fd566-6acd-4909-888c-593e7b59b2df">

**Comment by HoussemNasri** [comment]
> Isn't this resolved thanks to scite.ai? Refs https://github.com/koppor/jabref/issues/375 and https://github.com/JabRef/jabref/pull/10617

Not really, this issue is about displaying the total citations in the 'Citation relation' tab, getting the number of citations was already possible with the SematicScholar API, however, we now have 2 options.

**Comment by koppor** [comment]
Couldn't be it as simple as displaying the total number of citations in each heading?

![Image](https://github.com/user-attachments/assets/fb835361-abaa-41e6-824b-ca6824df921c)

---

I think, it would also be good to have an "intelligent" maintable column showing the number of citations (maybe using the cache of the citation relations)? --> another issue?

**Comment by koppor** [comment]
Duplicate of https://github.com/JabRef/jabref/issues/13477

(Sorry that I missed this while posting the other issue)
